crypto/tss/integration/tss_test.go

```diff
@@ -352,3 +497,73 @@ func (p *resharePeerManager) MustSend(id string, message proto.Message) {
 	msg := message.(types.Message)
 	Expect(d.AddMessage(msg)).Should(BeNil())
 }
+
+type addShareOldPeerManager struct {
```

Comment:
Maybe we can merge dkgManager, signerManager, reshareManager and addShareOldPeerManager into one?